src/nanopolish_read_db.cpp

```diff
@@ -292,3 +312,22 @@ void ReadDB::print_stats() const
     }
     fprintf(stderr, "[readdb] num reads: %zu, num reads with path to fast5: %zu\n", m_data.size(), num_reads_with_path);
 }
+
+bool ReadDB::get_slow5_mode() const {
+    return slow5_mode;
+}
+
+void ReadDB::close_db() const {
```

Comment:
I am not very good at OOP concepts, but I think this thing can go into the destructor of readDB so it does not need to be manually called?

Reply:
Thanks for pointing this out. This code is now moved inside ReadDB destructor.

Reply:
Agree, this should be in the destructor. Good catch and thanks for fixing it